src/lib/throttle.js

```diff
@@ -132,3 +136,55 @@ export function getReturnToken() : ?string {
         return token;
     }
 }
+
+type CustomThrottle = Throttle & {
+    isActive : () => boolean
+};
+
+export function buildFundingLogoThrottle(props : Object) : CustomThrottle {
+
+    let emptyThrottle = {
```

Comment:
Seems like rather than creating an entire mock throttle here along with a `CustomThrottle` type, could just `return`, right? Then do a null check before using the throttle to check if it's active? Might simplify this case a little.

Reply:
Done.